packages/terra-infinite-list/src/terra-dev-site/test/infinite-list/InfiniteListSelection.test.scss

```diff
@@ -0,0 +1,7 @@
+@import './InfiniteListTestCommon';
+
+:local {
+  .item-content {
+    @include content-common(50px);
```

Comment:
lets drop the @include syntax and directly add the styles here

Reply:
Done